tests/catalyst/tools/test_registry.py

```diff
@@ -99,7 +102,7 @@ def test_from_config():
     assert res == {"a": 1, "b": 2}
 
     res = r.get_from_params(**{})
-    assert res is None
+    assert res == {}
```

Comment:
should we return `None` as the default non-parametrized answer?

Reply:
We can check for empty dicts and return `None` in that case. But by default `get_from_params` returns either instance or kwargs (in case it is not possible to create instance from params)

Reply:
> in case it is not possible to create an instance from params
btw, why do we need to return the kwargs?